rest/route/agent.go

```diff
@@ -1144,35 +1143,29 @@ func (h *servePatchHandler) Parse(ctx context.Context, r *http.Request) error {
 	if h.taskID = gimlet.GetVars(r)["task_id"]; h.taskID == "" {
 		return errors.New("missing task ID")
 	}
-	if patchParam, exists := r.URL.Query()["patch"]; exists {
-		h.patchID = patchParam[0]
-	}
 	return nil
 }
 
 func (h *servePatchHandler) Run(ctx context.Context) gimlet.Responder {
-	if h.patchID == "" {
-		t, err := task.FindOneId(ctx, h.taskID)
-		if err != nil {
-			return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "finding task '%s'", h.taskID))
-		}
-		if t == nil {
-			return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
-				StatusCode: http.StatusNotFound,
-				Message:    fmt.Sprintf("task '%s' not found", h.taskID),
-			})
-		}
-		h.patchID = t.Version
+	t, err := task.FindOneId(ctx, h.taskID)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "finding task '%s'", h.taskID))
+	}
+	if t == nil {
+		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
+			StatusCode: http.StatusNotFound,
+			Message:    fmt.Sprintf("task '%s' not found", h.taskID),
+		})
 	}
 
-	p, err := patch.FindOne(ctx, patch.ByVersion(h.patchID))
+	p, err := patch.FindOne(ctx, patch.ByVersion(t.Version))
```

Comment:
The h.patchID was always set to t.Version, I just removed the field since it didn't provide any value now